CHANGELOG.md

```diff
@@ -47,15 +49,22 @@ New Features
   feature in twilio-video.js@2.1.0, followed by removing our existing SIP-based
   signaling transport altogether in twilio-video.js@2.2.0.
 
-  **NOTE:** The new signaling transport will reject access tokens containing configuration
-  profiles, which were deprecated when we [announced](https://www.twilio.com/blog/2017/04/programmable-video-peer-to-peer-rooms-ga.html#room-based-access-control)
-  the general availability of twilio-video.js@1.0.0. Use the [Programmable Video REST API](https://www.twilio.com/docs/video/api)
-  if you want to override the default settings while creating a Room.
-
-- This version of twilio-video.js will support Safari 12.1 and above which enables
-  Unified Plan as the default SDP format. We highly recommend that you upgrade your
-  twilio-video.js dependency to this version so that your application continues to
-  work on Safari versions 12.1 and above. (JSDK-2306)
+  **NOTE:** The new signaling transport will:
+  - disconnect you from the Room with an [AccessTokenExpiredError](https://www.twilio.com/docs/api/errors/20104)
+    if while reconnecting, it detects that your AccessToken is expired. Therefore,
+    we recommend that you create an AccessToken that is valid for the maximum
+    allowed session duration, i.e., *4 hours*.
```

Comment:
Link to docs that show where the ttl field is.

Reply:
Can we document the 4 hour Participant limit as part of the TCMP rollout, and link to that page? For example:

Therefore, we recommend that you create an AccessToken that is valid for the [maximum allowed session duration]() (currently 4 hours).

Reply:
@ceaglest 
I agree. I'm working on a wagtail tutorial for Network Handoff. However, it will take some time to be published since it has to go through a process of review and get published. I don't want to hold off this release because of that.

Reply:
That's understandable, would it be possible to update the text without a link for now?

I don't want to imply that the 4 hour limit will never be raised if we are able to remove this infra side limitation in the future.